src/Microsoft.TestPlatform.Common/Utilities/RunSettingsProviderExtensions.cs

```diff
@@ -95,7 +95,7 @@ public static void UpdateRunSettingsNode(this IRunSettingsProvider runSettingsPr
         /// <returns></returns>
         public static Match GetTestRunParameterNodeMatch(this IRunSettingsProvider runSettingsProvider, string node)
         {
-            var attrName = $"(?<{AttributeNameString}>\\w+)";
+            var attrName = $"(?<{AttributeNameString}>[\\w.:-]+)";
```

Comment:
No need to do this but converting this and the line below to $@ string would make it so much easier to read.